fix more E2E tests

sroet opened the pull request:
This will become a continuation of #123

sroet commented:
@McHaillet I have encountered a test which differged so much from the script that I don't know what the option mapping is anymore:
```python
outfile = 'OutputCCC/ellipse.mrc'
func = 'create_mask.py'
cmd = f"{func} -e {128} -m {40} -n {30} -l {20} -o {outfile} -s {5} -c {3}"
```
errors out with:
```
Some exception(s) while parsing the command line arguments:
Could not match argument -e to any option
Could not match argument 128 to any option
Could not match argument -l to any option
Could not match argument 20 to any option
Option -b requires arguments but none are given
Option -r requires arguments but none are given

Use '--help' to get information on how this script can be called.
```
Do you know what the current correct flags are for that script?

McHaillet commented:
Should have used `git mv` when I renamed that file :0

These are the changed parameters:
`-e` should be `-b` or `--boxSize`
`-m` should be `-r` or `--radius`
`-n` should be `-m` or `--minor1`
`-l` should be `-n` or `--minor2`

So the command should be like this:
`f"{func} --boxSize {128} --radius {40} --minor1 {30} --minor2 {20} --outputFileName {outfile} --sigma {5} --cutoff {3}"`

sroet commented:
> Should have used `git mv` when I renamed that file :0
> 
> These are the changed parameters: `-e` should be `-b` or `--boxSize` `-m` should be `-r` or `--radius` `-n` should be `-m` or `--minor1` `-l` should be `-n` or `--minor2`
> 
> So the command should be like this: `f"{func} --boxSize {128} --radius {40} --minor1 {30} --minor2 {20} --outputFileName {outfile} --sigma {5} --cutoff {3}"`

`--minor1` was not a recognized keyword (it was `--minor`) and neither was `--cutoff` (it was `--cuttoff` 🤦‍♂️ ) so I updated the keywords in the script (not the test) so that your command would work

sroet commented:
@McHaillet please review this PR, I think it is best to merge this in (now that test_BinScripts works) and continue on a separate branch/PR to keep review manageable

McHaillet reviewed (approved):
Looks good!
